cmd/mistryd/job.go

```diff
@@ -233,6 +252,20 @@ func (j *Job) StartContainer(ctx context.Context, cfg *Config, c *docker.Client,
 	return result.State.ExitCode, nil
 }
 
+var conflictRe = regexp.MustCompile(`Conflict. The container name "\S+" is already in use by container "(\S+)".`)
```

Comment:
I'm concerned that this is too fragile and non-future-proof. Can't we just check if the container exists and if it does, delete it?

Reply:
And they don't expose the helper or something we could reuse?

Reply:
I switched to a `renameIfExists` before creation. I'm renaming, I'm not sure we want to disrupt any existing containers. Any rogue containers will be garbage collected with our periodic prune